src/components/MewAddressSelect/MewAddressSelect.vue

```diff
@@ -0,0 +1,361 @@
+<template>
+  <!-- ===================================================================================== -->
+  <!-- Mew Address Select -->
+  <!-- ===================================================================================== -->
+  <v-combobox
+    ref="mewAddressSelect"
+    v-model="addressValue"
+    height="62"
+    class="address-select pa-0 rounded-lg"
+    color="primary"
+    :items="items"
+    :label="label"
+    item-value="address"
+    item-text="address"
+    :placeholder="placeholder"
+    :disabled="disabled"
+    :error-messages="errorMessages"
+    :hint="hint || resolvedAddr || ''"
+    :persistent-hint="resolvedAddr.length > 0"
```

Comment:
Can you change this to :persistent-hint="resolvedAddr.length > 0 || hint.length > 0"? So the hints can also be persistent